### ISSUE TITLE
urlscan version in pypi has a bad shebang

### ISSUE THREAD

**Issue by ElDeveloper** [firecat53/urlscan#24]
I just installed from pypi and got the following:

```bash
yoshikivazquezbaeza:~$ pip install urlscan
You are using pip version 6.1.1, however version 7.1.0 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
Collecting urlscan
  Using cached urlscan-0.8.0-py2.py3-none-any.whl
Requirement already satisfied (use --upgrade to upgrade): urwid in /Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages (from urlscan)
Installing collected packages: urlscan
Successfully installed urlscan-0.8.0
yoshikivazquezbaeza:~$ urlscan
-bash: /Library/Frameworks/Python.framework/Versions/2.7/bin/urlscan: /home/firecat53/.local/share/virtualenvs/urlscan/bin/python: bad interpreter: No such file or directory
yoshikivazquezbaeza:~$ head `which urlscan`
#!/home/firecat53/.local/share/virtualenvs/urlscan/bin/python
#
# A simple urlview replacement that handles things like quoted-printable
# properly.  aka "urlview minus teh suck"
#
#   Copyright (C) 2006-2007 Daniel Burrows
#   Copyright (C) 2014 Scott Hansen
#
#   This program is free software; you can redistribute it and/or
#   modify it under the terms of the GNU General Public License as
```

Which is fixed by changing the shebang, in my case to `/usr/bin/env python`.

**Comment by firecat53** [comment]
Thanks, not sure how that happened! Should be fixed with the 0.8.1 release. Let me know if it installs correctly now.

Scott

**Comment by ElDeveloper** [comment]
It is fixed, thanks @firecat53!
